src/pybind/mgr/mgr_module.py

```diff
@@ -566,6 +568,28 @@ def get_daemon_status(self, svc_type, svc_id):
         """
         return self._ceph_get_daemon_status(svc_type, svc_id)
 
+    def mon_command(self, cmd_dict):
```

Comment:
Hm. Looks very similar to `dashboard.services.ceph_service.CephService#send_command`